### ISSUE TITLE
Compiling qwerty on MacOS

### ISSUE THREAD

**Issue by jiimaho** [sunaku/glove80-keymaps#20]
Hello,

First of all thanks for a great setup! I really like it and would love to get it working with qwerty but something is failing.

I'm on MacOS and trying to compile with ./rake using Docker.

How to reproduce:
1. Clone repo
2. Inside rake file, add `--user $(id -u):$(id -g)` to the docker run command (otherwise it throws permission denied errors)
3. Start docker
4. Run ./rake
5. At least keymap.dtsi shows as modified in git (this is good I guess)

Actual:
* keymap.dtsi ends with a **very weird hack**, see picture 
<img width="603" alt="image" src="https://github.com/user-attachments/assets/1f499c56-a7f2-4190-b1d8-47adbcba29ed">
* content of keymap.dtsi do not work when building firmware at https://my.glove80.com/#/edit

Expected:
* keymap.dtsi to be valid
* content of keymap.dtsi to be uploaded and firmware to be built fine.

**Comment by sunaku** [comment]
That hack line at the end of the generated snippet is a normal part of the source code.  Have you modified anything else in your clone of the repository besides the `docker run` command?  For example, running `./rake` should ultimately produce the same `keymap.dtsi` file as that which is already checked into the Git repository -- so there should be no differences reported by `git diff` after regeneration.

**Comment by jiimaho** [comment]
Thanks for a quick reply!

Yea I saw that that the hack line was included when loading your [latest release](https://my.glove80.com/#/layout/user/f4372744-fce9-4678-ac4b-3d8ef8768160) too. Thanks for clarifying, now I know that is expected.

I totally forgot to mention that I moved layers too. Edited original question and added that as step 3 in bold.

**Comment by jiimaho** [comment]
Hm, I tried it all again just now, also changed OS to be  `#define OPERATING_SYSTEM 'M' // macOS`
I was prepared to take a print screen of the error I previously got on the moergo site, but now it built the firmware... :) 

Let me check if that works and maybe I can close this issue then.

**Comment by sunaku** [comment]
Yes, the symbol layer should work the same irrespective of the alpha layout.  Are you using the en-US locale?

**Comment by jiimaho** [comment]
Yes using the en-US locale for now, even though I will eventually get it to work with sv-SE too or fix my swedish characters somehow.

**Comment by emillinden** [comment]
> Yes using the en-US locale for now, even though I will eventually get it to work with sv-SE too or fix my swedish characters somehow.
> 
> But yea I haven't changed much really, so a bit strange the symbol layer is wrong. Will have to try again tomorrow.

@jiimaho Any luck on getting ÅÄÖ keys working? Namely the `&RightPinky (SEMI, LAYER_QWERTY)` key. I did try replacing `SEMI` with `SV_OE` and `Ö` but both results in a build error regardless of locale (tried en-US and sv_SE).

**Comment by sunaku** [comment]
I have removed the punctuation keycode overrides (such as `#define DLLR` for dollar sign) starting with the v40 release, so you should be able to use your preferred locale and redefine those keycodes accordingly.  See @helto4real's keymap at https://github.com/helto4real/glove80-keymaps for an example of how to implement this, especially for the Swedish (sv-SE) locale.
